src/main/java/net/apnic/rdap/rdap/RDAPObjectFactory.java

```diff
@@ -0,0 +1,93 @@
+package net.apnic.rdap.rdap;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Factory helper class for constructing conformant RDAP objects for client
+ * requests that are contextualised and pre filled with default notices.
+ */
+public class RDAPObjectFactory
+{
+    private static final RDAPConformance DEFAULT_CONFORMANCE =
+        RDAPConformance.LEVEL_0;
+
+    private List<RDAPNotice> defaultNotices = null;
```

Comment:
(Suggestion) Can you make this immutable (unmodifiable list / immutable RDAPNotice)? then you won't need to copy it.